src/main/java/fi/aalto/cs/apluscourses/ui/exercise/ExercisesView.java

```diff
@@ -66,6 +68,11 @@ private void createUIComponents() {
     exerciseGroupsTree.setCellRenderer(new ExercisesTreeRenderer());
     exerciseGroupsTree.addNodeAppliedListener(
         ActionUtil.createOnEventLauncher(OpenItemAction.ACTION_ID, exerciseGroupsTree));
+
+    new TreeSpeedSearch(exerciseGroupsTree, x -> {
```

Comment:
Nitpick: can we rename `x` to `path` (or `treePath`)

Reply:
Yes